doc_template/examples_root/examples/cell_types_ex1.py

```diff
@@ -3,7 +3,8 @@
 ct = CellTypesApi()
 
 # a list of dictionaries containing metadata for cells with reconstructions
-cells = ct.list_cells(require_reconstruction=True)
+cells = ct.list_cells_api(require_reconstruction=True)
+print(cells[0])
```

Comment:
Is this a lingering debug print? I don't think that the inline examples get run by sphinx (and I don't think this one gets included at all).

Reply:
good catch, thanks.